src/libs/ajax.js

```diff
@@ -944,6 +944,16 @@ const DataRepo = signal => ({
     //TODO: Update this link to hit the real endpoint
     const res = await fetchRawls(`dunno/what/this/is/${id}/requestAccess`, _.merge(authOpts(), { signal }))
     return res.json()
+  },
+
+  getPreviewMetadata: async id => {
```

Comment:
Do these need to be feature flagged?

Reply:
According to FB, only the preview table endpoint needs to be feature flagged. The convention for the feature flagging has been to catch it in the part of the code that runs the feature (and not here in the ajax.js file). And the part of the preview page that calls the previewEndpoint is wrapped in a "withErrorReporting" method that will catch the error. Then the page itself should be hidden from view (unless they manage to type it into the search bar correctly, and even so, it should still catch the error and report it to the user)

Is there any further action that I should do?